src/database/adapters/__init__.py

```diff
@@ -220,6 +215,10 @@ async def set_user_github_config(user: KhojUser, pat_token: str, repos: list):
     return config
 
 
+def get_or_create_search_model():
+    return SearchModel.objects.filter().get_or_create()[0]
```

Comment:
This throws an error when there are multiple SearchModels.

Reply:
Including stack trace:
```python
[17:59:54] ERROR    🚨 Failed to configure server on app load:   configure.py:98
                    release unlocked lock
                    ╭─── Traceback (most recent call last) ────╮
                    │ /app/src/khoj/configure.py:124 in        │
                    │ configure_server                         │
                    │                                          │
                    │   121 │   │   state.search_models = conf │
                    │       state.config.search_type)          │
                    │   122 │   │   initialize_content(regener │
                    │   123 │   except Exception as e:         │
                    │ ❱ 124 │   │   raise e                    │
                    │   125 │   finally:                       │
                    │   126 │   │   state.config_lock.release( │
                    │   127                                    │
                    │                                          │
                    │ /app/src/khoj/configure.py:117 in        │
                    │ configure_server                         │
                    │                                          │
                    │   114 │   # Initialize Search Models fro │
                    │   115 │   try:                           │
                    │   116 │   │   state.embeddings_model =   │
                    │       EmbeddingsModel(get_or_create_sear │
                    │ ❱ 117 │   │   state.cross_encoder_model  │
                    │       CrossEncoderModel(get_or_create_se │
                    │   118 │   │                              │
                    │   119 │   │   state.config_lock.acquire( │
                    │   120 │   │   state.SearchType = configu │
                    │                                          │
                    │ /app/src/database/adapters/__init__.py:2 │
                    │ 19 in get_or_create_search_model         │
                    │                                          │
                    │   216                                    │
                    │   217                                    │
                    │   218 def get_or_create_search_model():  │
                    │ ❱ 219 │   return SearchModel.objects.fil │
                    │   220                                    │
                    │   221                                    │
                    │   222 class ConversationAdapters:        │
                    │                                          │
                    │ /usr/local/lib/python3.10/dist-packages/ │
                    │ django/db/models/query.py:916 in         │
                    │ get_or_create                            │
                    │                                          │
                    │    913 │   │   # to avoid potential tran │
                    │    914 │   │   self._for_write = True    │
                    │    915 │   │   try:                      │
                    │ ❱  916 │   │   │   return self.get(**kwa │
                    │    917 │   │   except self.model.DoesNot │
                    │    918 │   │   │   params = self._extrac │
                    │    919 │   │   │   # Try to create an ob │
                    │                                          │
                    │ /usr/local/lib/python3.10/dist-packages/ │
                    │ django/db/models/query.py:640 in get     │
                    │                                          │
                    │    637 │   │   │   raise self.model.Does │
                    │    638 │   │   │   │   "%s matching quer │
                    │    639 │   │   │   )                     │
                    │ ❱  640 │   │   raise self.model.Multiple │
                    │    641 │   │   │   "get() returned more  │
                    │    642 │   │   │   % (                   │
                    │    643 │   │   │   │   self.model._meta. │
                    ╰──────────────────────────────────────────╯
                    MultipleObjectsReturned: get() returned more
                    than one SearchModel -- it returned 2!

                    During handling of the above exception,
                    another exception occurred:

                    ╭─── Traceback (most recent call last) ────╮
                    │ /app/src/khoj/configure.py:96 in         │
                    │ initialize_server                        │
                    │                                          │
                    │    93                                    │
                    │    94 def initialize_server(config: Opti │
                    │    95 │   try:                           │
                    │ ❱  96 │   │   configure_server(config, i │
                    │    97 │   except Exception as e:         │
                    │    98 │   │   logger.error(f"🚨 Failed t │
                    │    99                                    │
                    │                                          │
                    │ /app/src/khoj/configure.py:126 in        │
                    │ configure_server                         │
                    │                                          │
                    │   123 │   except Exception as e:         │
                    │   124 │   │   raise e                    │
                    │   125 │   finally:                       │
                    │ ❱ 126 │   │   state.config_lock.release( │
                    │   127                                    │
                    │   128                                    │
                    │   129 def initialize_content(regenerate: │
                    │       init=False, user: KhojUser = None) │
                    ╰──────────────────────────────────────────╯
                    RuntimeError: release unlocked lock
```

Reply:
Fixed by Saba